tests/parsers/thermo_fisher_nanodrop_eight/testdata/Thermo_NanoDrop_Eight_example01.json

```diff
@@ -7,7 +7,7 @@
                     "measurement time": "2020-05-26T15:36:00+00:00",
                     "measurement document": [
                         {
-                            "measurement identifier": "1fc444d9-389e-4df9-8a82-54460605f853",
+                            "measurement identifier": "f692658e-62f2-4383-8770-f0d97d2c17df",
```

Comment:
Hopefully #296 will merge before this, in which case none of these ids will change anymore!